otwcode/otwarchive
# AO3-5830 Tag search: Allow searching by fandom, searching for non-canonical tags, and sorting results

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-5830

## Purpose

1. Create new Tag Search form with sorting and searching by fandom
2. Update and create new tests 
3. Remove old form and update Browse Tags page

## Testing Instructions

- Try searching by tag name, tag type, wrangling status 
- Try sorting by uses and kudos
- Try filtering by fandom(s) (Test for single, multiple, non-existent and non-canonical fandoms)
- Check the Browse Tags page for new description of tag cloud and link to Search Tags

## References

## Credit

weeklies (she/her)
